Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/contests/contest-edit/ContestEdit.tsx

```diff
@@ -85,7 +86,7 @@ const ContestEdit = (props:IContestEditProps) => {
 
     const { data, isLoading } = useGetContestByIdQuery(
         { id: Number(contestId) },
-        { skip: !isEditMode || (isEditMode && !currentContest === undefined) },
+        { skip: !isEditMode || skipGettingContest },
```

Comment:
We cannot use only one of the booleans since this component is called on different places.  First it is called for contest creation this means  we will use isEditMode. Second it is used in Quick edit modal, there the isEditMode = true, skipGettingContest = false, Third: it is called in the ContestAdministrationPage , then isEditMode = true but also skipGettingContest = true because the same call is made in the AdministrationContestPage, because some of the data is needed for other components inside AdministrationContestPage. This can be achieved by event propagation, but it is not good idea to do it since it is more complicated than using one property to determine if the query must be executed or not.

Reply:
Fixed.